### ISSUE TITLE
How to get url with options from this library

### ISSUE THREAD

**Issue by RobinJayaswal** [filestack/filestack-js#130]
Say I have this code:

```
const result = await filestack.upload(imagePath, { onRetry }, { filename }, token)

return result.url
```

It'll give me `https://cdn.filestackcontent.com/{handle}`. If I want to return the url but with a max height and compression turned on, do I need to parse and reconstruct this url myself (and assume the handle is the final element in the route)? Or is there an easier way?

**Comment by velveteer** [comment]
@RobinJayaswal Have you seen the client `transform` method? You can pass it a handle (in your case `result.handle`) and an object of transform options for constructing a new URL: https://filestack.github.io/filestack-js/classes/client.html#transform

**Comment by RobinJayaswal** [comment]
@velveteer thanks for the response, exactly what I was looking for! 

One quick follow-up. I am trying to use the `compress` option, but I keep getting `Error: Wrong options provided: Invalid value true supplied to /compress: Nil`. 

I see that in urls for transformations, compress has no parameters (https://cdn.filestackcontent.com/compress/6fWKQk3Tq2numL2dACWw), but I do not see how I can accomplish that with this function. I have tried:

```
      compress: true,
      compress: {},
      compress: null,
```
And all throw.

**Comment by RobinJayaswal** [comment]
Nevermind, above error fixed by upgrading to 1.4. Thanks!
